sdk/search/Azure.Search.Documents/src/Indexes/Models/PiiDetectionSkill.cs

```diff
@@ -21,9 +21,9 @@ public partial class PiiDetectionSkill
             set
             {
                 _maskingMode = value;
-                if ((_maskingMode == PiiDetectionSkillMaskingMode.Replace) && (MaskingCharacter == null))
+                if ((_maskingMode == PiiDetectionSkillMaskingMode.Replace) && (Mask == null))
                 {
-                    MaskingCharacter = "*";
+                    Mask = "*";
```

Comment:
Does this make sense to set for the user if they forget to set the value?

Reply:
I don't know why @Mohit-Chakraborty has added this condition [here](https://github.com/Azure/azure-sdk-for-net/pull/22455/commits/f969e7ef63f89cdac32f2a457d5a036af516551a#diff-16312b54dcc119f23189882ce77d7ba07ba8f5ea4c4701779d6bc15c32b628ae). I can see one comment regarding this skill [here](https://github.com/Azure/azure-sdk-for-net/pull/22455#discussion_r665547010) but it doesn't mention about the condition.

@tg-msft do you remember why do we need [this](https://github.com/Azure/azure-sdk-for-net/blob/ca79f179671da929a80bf50baac7318b3ec0d223/sdk/search/Azure.Search.Documents/src/Indexes/Models/PiiDetectionSkill.cs#L24-L27) condition here?

Reply:
Oh wait, this has never GAed before?  Maybe we could remove it for safety's sake as there's been a decent amount of evolution in the AI Language space lately.  We should sanity check that the service gives a clear, diagnosable error message though if it's missing.

Reply:
Removed the condition, now we are relying on the service to handle all the scenarios.